vb-be/src/util/error.ts

```diff
@@ -1,14 +1,19 @@
+import { HttpException } from '@nestjs/common';
+
 export enum ErrorText {
   InvalidAuthState = 'Invalid authentication state',
   InvalidRTArg = 'Invalid argument for refresh token',
+  InvalidTokenPair = 'Invalid token pair',
   Generic = 'Something went wrong. Please try again later',
   TokenError = 'Cannot create tokens without user info',
   InvalidDataSet = 'Invalid dataset',
+  Unauthorized = 'Unauthorized',
 }
 
-export class ErrorHandler extends Error {
+// https://docs.nestjs.com/exception-filters
+export class ErrorHandler extends HttpException {
```

Comment:
nest uses a sort of innate error handler.  We can still use our system, we just had to change the class like this

Reply:
ah cool